skins/cat17/src/scripts/supporters.js

```diff
@@ -0,0 +1,38 @@
+(function( $ ) {
+	$( document ).ready( function() {
+		initSupportersTable();
+		attachEvents();
+	} );
+
+	function initSupportersTable() {
+		$( '.donors tbody tr' ).each( function() {
+			if( getComment( this ) ) {
+				$( this ).addClass( 'commented' );
+			}
+		} );
+	}
+
+	function attachEvents() {
+		var commentedRows = $( '.commented' );
+		commentedRows.click( function() {
```

Comment:
/me is TRIGGERED by avoidable variable

Reply:
Nope, this was an oversight, thanks for pointing it out.